doc/whats_new/v0.24.rst

```diff
@@ -78,19 +81,15 @@ Changelog
   samples between the train and test set on each fold.
   :pr:`13204` by :user:`Kyle Kosic <kykosic>`.
 
+:mod:`sklearn.feature_selection`
+................................
+
```

Comment:
This section (and its contents) should be higher up so that the modules are in alphabetical order.